src/QMCHamiltonians/HamiltonianFactory.h

```diff
@@ -30,39 +30,14 @@ class HamiltonianFactory : public MPIObjectBase
 {
 public:
   typedef std::map<std::string, ParticleSet*> PtclPoolType;
-  typedef std::map<std::string, WaveFunctionFactory*> OrbitalPoolType;
-
-  ///type of the lattice. 0=non-periodic, 1=periodic
-  int PBCType;
-  ///target ParticleSet
-  ParticleSet* targetPtcl;
-  ///many-body wavefunction object
-  QMCHamiltonian* targetH;
-  ///reference to the PtclPoolType
-  PtclPoolType& ptclPool;
-  ///reference to the WaveFunctionFactory Pool
-  OrbitalPoolType& psiPool;
-  ///input node for a many-body wavefunction
-  xmlNodePtr myNode;
-
-  ///name of the TrialWaveFunction
-  std::string psiName;
-
-  ///list of the old to new name
-  std::map<std::string, std::string> RenamedProperty;
+  typedef std::map<std::string, WaveFunctionFactory*> PsiPoolType;
```

Comment:
prefer 
```c++
using PsiPool = std::map<std::string, WaveFunctionFactory*>;
```

Although considering the existing typedefs perhaps a cleanup stage 2.

Reply:
Will change it when I resolve conflict with a later PR.